### ISSUE TITLE
Fedora 35 detected as OracleLinux

### ISSUE THREAD

**Issue by juhp** [stanislav-tkach/os_info#275]
I am trying the `os_info` cli on Fedora Linux 35 (which is still in development: due to be released in October).
I ran `cargo install os_info_cli` on Fedora 35 x86_64 and then when I run it:

```ShellSession
$ cargo install os_info_cli
    Updating crates.io index
  Downloaded os_info_cli v1.0.2
  Downloaded 1 crate (7.1 KB) in 2.67s
  Installing os_info_cli v1.0.2
  :
  :
$ os_info 
OS information:
Type: OracleLinux
Version: 35.0.0
Bitness: 64-bit
```
Also here is the os-release contents:
```ShellSession
$ head /etc/os-release 
NAME="Fedora Linux"
VERSION="35 (Workstation Edition Prerelease)"
ID=fedora
VERSION_ID=35
VERSION_CODENAME=""
PLATFORM_ID="platform:f35"
PRETTY_NAME="Fedora Linux 35 (Workstation Edition Prerelease)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:35"
```
and rust version:
```ShellSession
$ rustc --version
rustc 1.54.0 (Fedora 1.54.0-1.fc35)
```

**Comment by stanislav-tkach** [comment]
Sorry for the terrible late reply! I have tried to fix (https://github.com/stanislav-tkach/os_info/pull/281) this issue. Could you please check the version from the master branch? I'm going to release a new version soon if the issue is fixed.

**Comment by bsherman** [comment]
I'm not the OP, but I recently installed via `cargo install os_info` which installed  `v1.0.2`.

I saw the same problem, so I came to see if there was a bug filed.

I also cloned the master branch which seems to still be 1.0.2... and it has the same problem, so I'd say this is not fixed.

Upon closer inspection, it seems your previous fix did work but only if the system had the `lsb_release` binary, which is not installed by default on Fedora.

So, this is falling back to the `file_release` branch and missing your fix in the `lsb_release` branch.

I've fixed a submitted PR, #293 .
